quill-core/src/main/scala/io/getquill/naming/NamingStrategy.scala

```diff
@@ -80,11 +80,15 @@ trait CamelCase extends NamingStrategy {
 }
 object CamelCase extends CamelCase
 
+trait PostgresEscape extends Escape {
+  override def column(s: String) = if (s.startsWith("$")) s else default(s)
```

Comment:
because `AbstractMethodError`
```
java.lang.AbstractMethodError: io.getquill.naming.NamingStrategySpec$$anonfun$9$$anon$1.io$getquill$naming$PostgresEscape$$super$column(Ljava/lang/String;)Ljava/lang/String;
[info]   at io.getquill.naming.PostgresEscape$class.column(NamingStrategy.scala:84)
[info]   at io.getquill.naming.NamingStrategySpec$$anonfun$9$$anon$1.column(NamingStrategySpec.scala:90)
[info]   at io.getquill.naming.NamingStrategySpec$$anonfun$9$$anonfun$apply$mcV$sp$10.apply$mcV$sp(NamingStrategySpec.scala:93)
```